pkg/pluginmanager/manager.go

```diff
@@ -697,24 +697,28 @@ func GetPluginGroup(groupIDAndVersion string, options ...PluginManagerOptions) (
 	return pg, nil
 }
 
-func logPluginInstallationMessage(p *discovery.Discovered, version string, isPluginInCache, isPluginAlreadyInstalled bool) {
+func getPluginInstallationMessage(p *discovery.Discovered, version string, isPluginInCache, isPluginAlreadyInstalled bool) (installingMsg, installedMsg string) {
 	withTarget := ""
 	if p.Target != configtypes.TargetUnknown {
-		withTarget = fmt.Sprintf("with target '%v' ", p.Target)
+		withTarget = fmt.Sprintf("with target '%v'", p.Target)
 	}
 
 	if isPluginInCache {
 		if !isPluginAlreadyInstalled {
-			log.Infof("Installing plugin '%v:%v' %v(from cache)", p.Name, version, withTarget)
+			installingMsg = fmt.Sprintf("Installing plugin '%v:%v' %v (from cache)", p.Name, version, withTarget)
```

Comment:
By putting the space before `(from cache)` instead of in `withTarget`, you will end up with a double space when the `withTarget` string is empty.

Reply:
Agree, fixed.